lib/fluent/plugin/in_s3.rb

```diff
@@ -181,6 +183,11 @@ def run
         @poller.poll(options) do |message|
           begin
             body = Yajl.load(message.body)
+            if @assume_sns
+              log.debug(body)
+              next unless body["Message"]
+              body = Yajl.load(body["Message"])
+            end
```

Comment:
How about this? Besides, Can you delete `log.debug(body)` at L184? it's duplicated.
```suggestion
            log.debug(body)
            if @assume_sns
              next unless body["Message"]
              body = Yajl.load(body["Message"])
            end
```